src/main/webapp/app/assessment/assessment-header/assessment-header.component.ts

```diff
@@ -93,8 +93,12 @@ export class AssessmentHeaderComponent {
         // if there is no 'save' button
         if (this.result?.completionDate) {
             return true;
+        } else if (Result.hasNonEmptyAssessmentNote(this.result)) {
```

Comment:
For this particular case, we decided that this is enough, since you can still cancel the assessment at this stage. Once other feedback is added, saving should work again from the `else` case, and in case you already saved a assessment note without other feedback, you can still cancel since, since nothing is lost anyway.